system_metrics_collector/src/system_metrics_collector/periodic_measurement_node.hpp

```diff
@@ -90,9 +91,6 @@ class PeriodicMeasurementNode : public system_metrics_collector::Collector, publ
    */
   bool setupStop() override;
 
-  // todo implement on publish timer callback, check if we need to clear the window
```

Comment:
Was this implemented? If so I think I missed where.

Reply:
This was implemented in https://github.com/ros-tooling/system_metrics_collector/pull/16. I'll put this comment back for this PR.

Reply:
I've put this comment back.